snakemake/parser.py

```diff
@@ -611,6 +611,22 @@ def args(self):
         )
 
 
+class Subworkflow_RuleLike(Script):
+    start_func = "@workflow.subworkflow"
+    end_func = "subworkflow"
+
+    def args(self):
+        # basedir
+        yield ", {!r}".format(os.path.abspath(os.path.dirname(self.snakefile.path)))
```

Comment:
f-strings are faster and more concise

Reply:
Absolutely!

I used `format` because other classes also use this style (`CWL`, `Script`, ...).
If wanted, I could change all of them to use f-strings.